ikarus/finiteelements/physicshelper.hh

```diff
@@ -56,22 +58,46 @@ namespace Ikarus {
   }
 
   /**
-   * \brief Traits for handling local views.
+   * \brief Traits for handling finite elements.
    *
-   * \tparam LocalView Type of the local view.
+   * \tparam Basis_ The basis type for the finite element.
+   * \tparam FERequirements_ The requirements for the finite element.
    * \tparam useRef Boolean indicating whether to use Eigen::Ref for VectorType and MatrixType.
    */
-  template <typename LocalView, bool useRef = false>
-  struct TraitsFromLocalView {
-    using GridEntity = typename LocalView::Element;
+  template <typename Basis_, typename FERequirements_, bool useRef = false>
+  struct TraitsFromFE {
+    /** \brief Convenience typedef alias for basis type */
```

Comment:
Can you use the same comment style as below. It is not clear what convenience you mean here. It is actually necessary.